commons/pdf-commons/src/main/java/se/idsec/signservice/security/sign/pdf/document/VisibleSignatureImage.java

```diff
@@ -56,8 +56,8 @@
 @Slf4j
 public class VisibleSignatureImage {
 
-  /** Basic date format used. */
-  public static final SimpleDateFormat BASIC_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm");
+  /** Default date format. */
```

Comment:
Well this comment isn't correct anymore? This is not the default. It's a variable that can be set.

Reply:
A small 'z'? Shouldn't it be 'Z'?

Reply:
I would suggest:
/** Default date format. */
public static final String DEFAULT_DATE_FORMAT = "yyyy-MM-dd HH:mm Z";

/** Date format for signing time. The default is {@link #DEFAULT_DATE_FORMAT}. */
private SimpleDateFormat dateFormat = new SimpleDateFormat(DEFAULT_DATE_FORMAT);

Reply:
No it should NOT be lage Z. That results in +0200. It must be small z, which results in "CEST" or similar.

Reply:
comments adressed in update